modules/library/referencing/src/test/python/proj/gt_proj_compare.py

```diff
@@ -0,0 +1,127 @@
+import csv
```

Comment:
As code part of GeoTools, they should have a copyright header nonetheless (they are not shipped, but neither are the java tests, which do have copyright headers).